analytics/scala/src/main/scala/com/koverse/example/spark/NaiveBayesPredictTransform.scala

```diff
@@ -39,7 +39,7 @@ class NaiveBayesPredictTransform extends JavaSparkTransform {
     val inputDataFrame:DataFrame = KoverseSparkSql.createDataFrame(context.getInputCollectionRdds.get(inputCollectionId),
       SQLContext , context.getInputCollectionSchemas.get(inputCollectionId))
 
-    val modelId = context.getInputCollectionIds.get(1)
+    val modelId = context.getInputCollectionIds.get(0)
     val modelRDD:JavaRDD[SimpleRecord] = context.getInputCollectionRdds.get(modelId)
```

Comment:
This is kind of the koverse 2.x pattern .. is this transform using one of the new 3.x SDK classes or is it still using a 2.x class?

Reply:
I didn't really change this one, so it is using the 2.10.x version. Maybe I should update it.